bundler/spec/dependabot/bundler/update_checker/conflicting_dependency_resolver_spec.rb

```diff
@@ -19,7 +19,7 @@
         "username" => "x-access-token",
         "password" => "token"
       }],
-      options: { bundler_2_available: bundler_2_available? }
+      options: { bundler_2_available: PackageManagerHelper.use_bundler_2? }
```

Comment:
Won't we need to change all specs to have `BUNDLED WITH 2` when this is true? Maybe worth adding an override just for tests until we've converted specs?

Reply:
Yeah, my plan is to add a temporary patch to [this method](https://github.com/dependabot/dependabot-core/pull/3350/files#diff-1f5350b0c5588ffb399b6384bb37abc35ea941b5897989c74f2ed70c83a0c6b6R27) that rewrites the lockfile to `BUNDLED WITH 2` when this is true and get the tests to call project fixtures via that.

This probably cannot merge until we've sorted out the project fixtures, but I wanted to lay out the helper logic + tests now since the behaviour has a few quirks around defaulting/failing back.

Reply:
Oh I should mention, that rewrite should eventually just be replaced with straight up forking the project fixtures for clarity since there may be other tweaks/changes.

Reply:
I've ended up adding the v2 project fixtures as part of this PR as it turned out easy to implement using this toggle to find the specs that failed due to missing fixtures. It might need broken out for review purposes, but the guts of it should be done.